api_implementation.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Which API does it support?",
+  "headers": ["API name", "url", "Implementation status"],
+  "headers attributes": ["name", "url", "implementation status"],
+  "data": [
+    {
+      "name": "User provisioning API",
+      "url":  "https://docs.nextcloud.com/server/14/admin_manual/configuration_user/user_provisioning_api.html",
```

Comment:
We have some URLs with NextCloud version and some without it. Therefore, I propose to record `date_last_checked` in the JSON, and display it in the table in a separate column.